pyproject.toml

```diff
@@ -57,12 +57,12 @@ lmfit = "^1.1.0"
 pandas = ">=2.1.0"
 psycopg2 = "^2.9.9"
 minio = "^7.2.5"
+sentry-sdk = ">= 1.45.0, < 2.0.0"
 
 
 [tool.poetry.group.dev.dependencies]
 black = "^22.12.0"
 compose-go = "^2.20.2"
-containers-sugar = "^1.9.3"
```

Comment:
any reason for removing containers-sugar from the dependencies?

Reply:
It's repeated. I removed from dev dependencies because it's on `tool.poetry.dependencies` already (see https://github.com/AlertaDengue/AlertaDengue/blob/enhancement/configure-sentry/pyproject.toml#L47).